CMakeLists.txt

```diff
@@ -53,6 +53,11 @@ ament_python_install_package(${PROJECT_NAME})
 # Install Python executables
 install(PROGRAMS
   karelics_vesc_can_driver/vesc_can_driver.py
+  karelics_vesc_can_driver/battery_status.py
+  karelics_vesc_can_driver/karelics_vesc_can_driver_utils.py
+  karelics_vesc_can_driver/max_current_monitor.py
+  karelics_vesc_can_driver/vesc.py
+  karelics_vesc_can_driver/vesc_messages.py
```

Comment:
Why are you installing these files?

Reply:
We added them because of a "modeul not found" error we were getting that we thought was coming from not adding them to the CMakeList, but now I have checked better and I was importing one of the modules wronly in the max_current_monitor.py so I removed the extra python source files besides just the vesc_can_driver.py